### PR TITLE
Moved the owner system to UUIDs. Changed all checks to UUIDs instead of ...

### LINKED ISSUE #258: [Suggestion] Convert the owner saving to UUIDs/GameProfiles to support name changes.
Maybe an IOwnableMachine interface and a little check in the EventListener (onPlayerPlacesBlock).

### PR DESCRIPTION
**Pull request by iTitus**
...display names. Closes #258.

### CODE REVIEW

File: src/main/java/buildcraftAdditions/blocks/multiBlocks/MultiBlockKEBT2.java [hunk 1]
```diff
@@ -42,8 +43,10 @@ public void onBlockPlacedBy(World world, int x, int y, int z, EntityLivingBase e
 		if (entity instanceof EntityPlayer) {
 			TileEntity tileEntity = world.getTileEntity(x, y, z);
 			if (tileEntity instanceof TileKineticEnergyBufferBase) {
-				((TileKineticEnergyBufferBase) tileEntity).setOwner(((EntityPlayer) entity).getDisplayName());
-				((TileKineticEnergyBufferBase) tileEntity).destroyer = ((EntityPlayer) entity);
+				EntityPlayer player = (EntityPlayer) entity;
+				TileKineticEnergyBufferBase keb = (TileKineticEnergyBufferBase) tileEntity;
+				keb.setOwner(PlayerUtils.getPlayerUUID(player));
+				keb.destroyer = (player);
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/blocks/multiBlocks/MultiBlockKEBT2.java]: why the ()?

File: src/main/java/buildcraftAdditions/blocks/multiBlocks/PaternTestingBlock.java [hunk 2]
```diff
@@ -25,4 +27,8 @@ public PaternTestingBlock() {
 	public void onBlockPlacedBy(World world, int x, int y, int z, EntityLivingBase entity, ItemStack stack) {
 		TestingPatern.build(new Location(world, x, y, z));
 	}
+
+	@Override
+	public void registerBlockIcons(IIconRegister register) {
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/blocks/multiBlocks/PaternTestingBlock.java]: why is this added? it's just an empty method

Comment (iTitus) [inline src/main/java/buildcraftAdditions/blocks/multiBlocks/PaternTestingBlock.java]: To prevent the exception shown in the console that the item icon couldnt be found.

File: src/main/java/buildcraftAdditions/blocks/multiBlocks/MultiBlockKEBT2.java (hunk 1 above)
Comment (iTitus) [inline src/main/java/buildcraftAdditions/blocks/multiBlocks/MultiBlockKEBT2.java]: Copy pasta.
